gettsim/benefits/arbeitsl_geld.py

```diff
@@ -61,9 +62,7 @@ def proxy_net_wage_last_year(
     prox_ssc = soz_vers_pauschale * max_wage
 
     # Fictive taxes (Lohnsteuer) are approximated by applying the wage to the tax tariff
-    prox_tax = eink_st_params["st_tarif"](
-        12 * max_wage - werbungs_pausch, eink_st_params
-    )
+    prox_tax = st_tarif(12 * max_wage - werbungs_pausch, eink_st_params)
```

Comment:
I'd argue for keeping `eink_st_params["st_tarif"]`. If we implement older tariffs with different functional forms, we have to revert this step.

Reply:
Its only two places in the code. If we really implement it, we can still change. But I think with implementing the DAG soon, our function assignment will change in gereal!